ambition_utils/rrule/forms.py

```diff
@@ -249,13 +249,33 @@ def save(self, **kwargs):
         """
         Saves the RRule model and returns it
         """
-        # Keep track if this is an existing rrule that needs occurrence updated
-        need_to_refresh_next_recurrence = False
+        # The next occurrence will be refreshed if changes to the object have been made.
+        need_to_refresh_next_occurrence = False
 
         # Get the rrule model from the cleaned data
         rrule_model = self.cleaned_data.get('rrule')
         if rrule_model:
-            need_to_refresh_next_recurrence = True
+            # Create sorted, arrays of values as strings for comparison.
+            form_rrule_params = sorted(
+                [str(value) for value in self.get_rrule_params_from_cleaned_data().values()]
+            )
+            original_rrule_params = sorted(
+                [str(value) for value in rrule_model.rrule_params.values()]
+            )
+
+            form_exclusion_params = sorted(
+                [str(value) for value in (self.cleaned_data.get('rrule_exclusion') or {}).values()]
+            )
+            original_exclusion_params = sorted(
+                [str(value) for value in (rrule_model.rrule_exclusion_params or {}).values()]
+            )
```

Comment:
I considered using `sorted(..., key=str)` but was not seeing `datetime` & `FakeDatetime` converted as expected. (The existing `distart` is a string but the submitted `distort` is an object.)

I don't love this and welcome suggestions.